doc/source/reference/routines.set.rst

```diff
@@ -20,6 +15,7 @@ Boolean operations
 .. autosummary::
    :toctree: generated/
 
+   ediff1d
```

Comment:
I don't think `ediff1d` belongs here with these other logical operators, `ediff1d` returns consecutive differences in the array.

Reply:
Sure! I created third section `Miscellaneous operations` with `ediff1d`

Reply:
I think it shouldn't appear in this file at all. There's already an entry for it in `routines.math.rst` in the "sums, products, differences" section of that file and it doesn't treat an array like a set, so it doesn't make sense to put it in the docs section for set operations.

Reply:
Sure! I removed it completely.